integration-tests/features/steps/common.py

```diff
@@ -213,44 +201,48 @@ def validate_analysis_result(context, ecosystem, package, version):
         jsonschema.validate(struct, schema)
 
 
+@when("I get a valid request ID")
 @then("I should get a valid request ID")
```

Comment:
I saw similar usage here: https://github.com/pkajaba/fabric8-analytics-common/blob/1e628e3f61beccf4510385a1f4b6722f89002891/integration-tests/features/steps/common.py#L155 so I assumed it's OK. I have to check it out